File: client/app/src/main/java/com/rh/heji/data/db/DealerDao.java
package com.rh.heji.data.db;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.OnConflictStrategy;
import androidx.room.Query;

import com.rh.heji.data.db.Dealer;

import java.util.List;

/**
 * Date: 2020/9/22
 * Author: 锅得铁
 * #
 */
@Dao
public interface DealerDao {
    @Insert(onConflict = OnConflictStrategy.IGNORE)
    void insert(Dealer user);

    @Query("select * from bill_dealer")
    List<Dealer> findAll();
}

File: client/app/src/main/java/com/rh/heji/data/db/DealerDao.kt
package com.rh.heji.data.db

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import androidx.room.Query

/**
 * Date: 2020/9/22
 * @author: 锅得铁
 * #
 */
@Dao
interface DealerDao {
    @Insert(onConflict = OnConflictStrategy.IGNORE)
    fun insert(user: Dealer?)

    @Query("select * from dealer")
    fun findAll(): List<Dealer>
}